src/arrays/multidimensional_arrays/hard/q3/CountsOdd.java

```diff
@@ -0,0 +1,26 @@
+package arrays.multidimensional_arrays.hard.q3;
+
+/*
+Make a Java program that initializes a multidimensional array in the main method and calls the countsOdd method. The
+countsOdd method should accept a multidimensional array of integers, count the number of odd integers in the array, and
+print the count.
+
```

Comment:
Change to:
"Make a Java program that asks the user for the values of a 3 by 3 multidimensional array in the main method and calls the countsOdd method. The countsOdd method should accept a multidimensional array of integers of any size, count the number of odd integers in the array, and print the count."